iOS Example Frame Carthage/iOS Example Frame Carthage.xcodeproj/project.pbxproj

```diff
@@ -292,6 +292,7 @@
 		957323EB286EF50F0068C115 /* applePayTokenInvalid.json in Resources */ = {isa = PBXBuildFile; fileRef = 95732393286EF50F0068C115 /* applePayTokenInvalid.json */; };
 		957323EC286EF50F0068C115 /* cardTokenInvalidNumber.json in Resources */ = {isa = PBXBuildFile; fileRef = 95732394286EF50F0068C115 /* cardTokenInvalidNumber.json */; };
 		957323ED286EF50F0068C115 /* applePayToken.json in Resources */ = {isa = PBXBuildFile; fileRef = 95732395286EF50F0068C115 /* applePayToken.json */; };
+		958826FC28902ED400385E7D /* UIViewExtension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 958826FB28902ED400385E7D /* UIViewExtension.swift */; };
```

Comment:
you can still move the content of this file into the existing file `UIViewExtensions.swift`. We don't need 2 files with extensions